lib/lure/repo.go

```diff
@@ -9,10 +9,23 @@ import (
 	"os/exec"
 	"reflect"
 	"strings"
+	"text/template"
 
 	"github.com/vsekhar/govtil/guid"
 )
 
+// Tprintf passed template string is formatted usign its operands and returns the resulting string.
+// Spaces are added between operands when neither is a string.
+// https://play.golang.org/p/COHKlB2RML
+func Tprintf(tmpl string, data map[string]interface{}) string {
+	t := template.Must(template.New("dummy").Parse(tmpl))
```

Comment:
Weird that you need a name absolutely. Is it because you are suppose to reuse the object instead of creating it for every commit?

Reply:
https://stackoverflow.com/questions/41176355/go-template-name

Looks like it just lets to refer to templates, e.g. using a getByName function or inside other templates, and they didn't want to make it optional.

Reply:
My use of dummy here could conflict if we started using templates elsewhere and used the same value. I should maybe make sure to use a unique name here?

Reply:
would be possible to store the template and reuse it? otherwise I'd put a guid yes